GoogleDataTransport/GDTTests/Common/Fakes/GDTTransportFake.m

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "third_party/firebase/ios/Releases/GoogleDataTransport/GDTTests/Common/Fakes/GDTTransportFake.h"
```

Comment:
Same as above, should be `import "GDTTests/Common/Fakes/GDTTransportFake.h"`

Reply:
Done in the following [commit.](https://github.com/firebase/firebase-ios-sdk/pull/3807/commits/f45f12f02a7c88dbb83b863270913bb43966b7b8)